theme/snippets/meta.liquid

```diff
@@ -11,17 +11,17 @@
 {% if template == 'index' %}
   <meta property="og:type" content="website">
   <meta property="og:title" content="INDEX">
-  <meta property="og:description" content="TO-DO: ADD DEFAULT HOME DESCRIPTION">
+  <meta property="og:description" content="Index is a community space promoting access, support and good vibrations within the creative disciplines.">
```

Comment:
Might be a good idea to move these defaults to our translation file, especially since there seems to be some duplication e.g. title, description, etc.